wowup-electron/src/app/addon-providers/github-addon-provider.ts

```diff
@@ -485,7 +495,14 @@ export class GitHubAddonProvider extends AddonProvider {
 
   private async getWithRateLimit<T>(url: URL | string): Promise<T> {
     try {
-      return await this._httpClient.get<T>(url.toString()).toPromise();
+      const personalAccessToken = await this._sensitiveStorageService.getAsync(PREF_GITHUB_PERSONAL_ACCESS_TOKEN);
```

Comment:
Might be worth looking into some form of encryption to not store it as plain text on the filesystem?

Reply:
Potentially, it wouldnt be super secure but something is better than nothing i suppose.